### PR TITLE
Fix an error

### PR DESCRIPTION
**Pull request by Ener-Getick**
see https://github.com/FriendsOfSymfony/FOSCommentBundle/pull/512#issuecomment-220064380

### CODE REVIEW

File: Constraints/LegacyCallbackHelper.php [hunk 1]
```diff
@@ -22,11 +22,15 @@
 {
     public function __construct($options = null)
     {
-        if($options['value'] == 'isValid' && !class_exists('Symfony\Component\Validator\Context\ExecutionContextInterface')) {
-            $options['value'] = 'isVoteValid';
+        if (isset($options['methods']) && is_array($options['methods'])) {
+            foreach ($options['methods'] as &$method) {
+                if($method === 'isValid' && !class_exists('Symfony\Component\Validator\Context\ExecutionContextInterface')) {
```
Comment (XWB) [inline Constraints/LegacyCallbackHelper.php]: Please add a space after the `if`.

File: composer.json [hunk 2]
```diff
@@ -60,8 +60,7 @@
         }
     },
     "autoload": {
-        "psr-0": { "FOS\\CommentBundle": "" }
+        "psr-4": { "FOS\\CommentBundle\\": "" }
```
Comment (XWB) [inline composer.json]: I'm not sure how this resolves the issue?

Comment (Ener-Getick) [inline composer.json]: oops shouldn't be there...

File: Constraints/LegacyCallbackHelper.php (hunk 1 above)
Comment (Ener-Getick) [inline Constraints/LegacyCallbackHelper.php]: good catch :+1: fixed

**Comment by XWB** [comment]
Thanks.

**Comment by Ener-Getick** [comment]
@XWB can we release a new stable version ?

**Pull request merged**